dashboard/test/ui/features/learning_platform/student_lesson_plan.feature

```diff
@@ -40,3 +40,6 @@ Feature: Student Lesson Plan
     And I wait until element "a:contains(All the Migrated Things)" is visible
     And I click selector "a:contains(All the Migrated Things)"
     And I wait until I am on "http://studio.code.org/s/allthemigratedthings?no_redirect=true"
+    And I wait until element "#ui-test-lesson-resources" is visible
+    And I click selector "#ui-test-lesson-resources"
+    Then I wait until I am on "http://localhost-studio.code.org:3000/s/allthemigratedthings?no_redirect=true"
```

Comment:
Shouldn't this go to a lesson not the script?

Reply:
oh shoot I copied it wronge

Reply:
oh shoot its passing for the wrong reasons grrrr

Reply:
Because the base branch isn't staging, I'm not sure Drone is running on it

Reply:
Jillian are you trying to test the button the script overview page here? I would add that at the beginning of the UI test. Instead of just navigating directly to the student lesson plan page I would navigate to the script page and then click the button and make sure you got to the student lesson plan page

Reply:
Yeah, I'm trying to test the button. I meant that it was passing locally for the wrong reasons. I'll move this logic to the front and try again